lib/websocket-terminal/src/main/go/server.go

```diff
@@ -75,6 +74,37 @@ func (wp *wsPty) Stop() {
 }
 
 func ptyHandler(w http.ResponseWriter, r *http.Request) {
+	tokenParam := r.URL.Query().Get("access_token")
```

Comment:
About authorship in repositories such as golang and docker they use authors file, so we need to do similar or something else and enter the agreement on this, what do you think @garagatyi ?